pkg/utils/utils.go

```diff
@@ -265,6 +267,38 @@ func GetPasswordStdin() (string, error) {
 	return strings.TrimSuffix(line, "\n"), nil
 }
 
+// GenerateRandomPassword generates random ASCII password
+func GenerateRandomPassword(length int) (string, error) {
```

Comment:
You can also use 1password's package to generate password - https://pkg.go.dev/go.1password.io/spg#hdr-Usage_overview